.github/workflows/dusk-tests.yml

```
@@ -0,0 +1,100 @@
+  workflow_dispatch:
+
+jobs:
+  dusk-tests:
+    runs-on: ubuntu-latest
+    
+    services:
+      # MySQL service
+      mysql:
+        image: mysql:8.0
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)